src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -115,34 +100,18 @@
 helps['account lock create'] = """
     type: command
     short-summary: Create a subscription lock.
-    examples:
-        - name: Create a read-only subscription level lock.
-          text: >
-            az account lock create --lock-type ReadOnly -n lockName
```

Comment:
On the other hand, removing this one for a strongly typed command does make sense because it is (or should be) obvious what you are trying to do.

Reply:
With the more specific lock commands available, is `lock create` even useful?

Reply:
No, but it would be a breaking change to remove. We added the more specific commands to improve the UX, but we still support the generic command.